http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -1,24 +1,22 @@
 package org.broadinstitute.dsde.workbench.leonardo
 package db
 
-import java.time.Instant
-
 import cats.syntax.all._
 import org.broadinstitute.dsde.workbench.google2.{DiskName, ZoneName}
 import org.broadinstitute.dsde.workbench.leonardo.SamResourceId.PersistentDiskSamResourceId
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.mappedColumnImplicits._
 import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.{dummyDate, unmarshalDestroyedDate}
 import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
-import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
+import java.time.Instant
 import scala.concurrent.ExecutionContext
 
 final case class PersistentDiskRecord(id: DiskId,
-                                      googleProject: GoogleProject,
+                                      cloudContext: CloudContext,
                                       zone: ZoneName,
                                       name: DiskName,
-                                      googleId: Option[GoogleId],
+                                      googleId: Option[ProxyHostName],
```

Comment:
should this still be called googleId?

Reply:
probably not...there're probably still cases here and there that Type and name don't exactly match, but I think we can fix over time....I'll fix this one